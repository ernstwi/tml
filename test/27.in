init foo
abc
foo -> baz
foo -> bar
