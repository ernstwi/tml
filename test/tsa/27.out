{
    "clocks": [],
    "edges": [
        {
            "from": "foo",
            "guard": null,
            "reset": null,
            "sync": null,
            "to": "bar"
        },
        {
            "from": "foo",
            "guard": null,
            "reset": null,
            "sync": null,
            "to": "baz"
        }
    ],
    "locations": [
        {
            "id": "abc",
            "initial": false,
            "invariant": null
        },
        {
            "id": "bar",
            "initial": false,
            "invariant": null
        },
        {
            "id": "baz",
            "initial": false,
            "invariant": null
        },
        {
            "id": "foo",
            "initial": true,
            "invariant": null
        }
    ]
}
